### ISSUE TITLE
Exam 4 Trace failed

### ISSUE THREAD

**Issue by theo-videira** [c-bertran/codeshell#2]
**Describe the bug**
I think Exam 4 always fail because code in [here](https://github.com/c-bertran/examshell/blob/main/exam/04/microshell/normal/microshell.c) can't compile

**Expected behavior**
Exam ok or another error.

**Screenshots**
```
=== TRACE ===
microshell.c:39:32: error: '&&' within '||' [-Werror,-Wlogical-op-parentheses]
        if (x < 2 || x > 2 || argv[1] && strcmp(argv[1], "-") == 0)
                           ~~ ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~
microshell.c:39:32: note: place parentheses around the '&&' expression to silence this warning
        if (x < 2 || x > 2 || argv[1] && strcmp(argv[1], "-") == 0)
                                      ^
                              (                                   )
1 error generated.
```
**Additional context**
None

**Comment by c-bertran** [comment]
Thanks for the report, the patch should arrive in a few minutes

**Comment by c-bertran** [comment]
The issue has been resolved in the new version 0.2.3, or on the last commit.
